src/pages/Dictionary/Dictionary.jsx

```diff
@@ -21,6 +25,18 @@ const Dictionary = () => {
 
   const status = getTermsEntriesStatus(termsStatus, entriesStatus);
 
+  const Text = ({ h1, h2, h3, className, children }) => {
+    const h1classname = h1 ? 'text-4xl font-extrabold text-yellow-200 text-left mb-3' : '';
+    const h2classname = h2 ? 'text-2xl font-bold text-left' : '';
+    const h3classname = h3 ? 'text-xl text-left ml-5' : '';
+    return (
+      <p className={`${h1classname} ${h2classname} ${h3classname} ${className}`}>{children}</p>
+    );
+  };
+
+  const [isLikeHovered, setIsLikeHovered] = useState(false);
```

Comment:
this state would make the whole Dictionary component get rerendered if a user hovers on the button i think.

Reply:
this hover feature should be passed to the icon props